sphinx_needs/directives/needimport.py

```diff
@@ -113,14 +113,13 @@ def run(self) -> Sequence[nodes.Node]:
             except (OSError, json.JSONDecodeError) as e:
                 # TODO: Add exception handling
                 raise SphinxNeedsFileException(correct_need_import_path) from e
-
-            errors = check_needs_data(needs_import_list)
-            if errors.schema:
-                logger.info(
-                    f"Schema validation errors detected in file {correct_need_import_path}:"
-                )
-                for error in errors.schema:
-                    logger.info(f"  {error.message} -> {'.'.join(error.path)}")
+        errors = check_needs_data(needs_import_list)
+        if errors.schema:
+            logger.info(
+                f"Schema validation errors detected in file {correct_need_import_path}:"
```

Comment:
ah actually `correct_need_import_path` is only available from the local file path if/esle branch, so this needs to now account for also pointing to the remote path